DashboardSettings: Fixes issue with tags list not updating changes are made (#32189)
grafana/grafana: docs/sources/administration/configuration.md

For example, given a cdn url like `https://cdn.myserver.com` grafana will try to load a javascript file from
`http://cdn.myserver.com/grafana-oss/v7.4.0/public/build/app.<hash>.js`.

### read_timeout

> Sets the maximum time using a duration format (5s/5m/5ms) before timing out read of an incoming request and closing idle connections.

Reviewer: ```suggestion
Sets the maximum time using the duration format (5s/5m/5ms) before reading of incoming request times out and idle connections close.
```

<hr>

## [annotations]

### cleanupjob_batchsize

> Configures the batch size for the annotation clean-up job. This setting is used for dashboard, API, and alert annotations.

Reviewer: ```suggestion
Configures the batch size for the annotation cleanup job. This setting is used for dashboard, API, and alert annotations.
```
